2025-10-14 15:58:05,192 [processing](INFO): Initializing backseat process (pilot lrauv based on marl)
2025-10-14 15:58:08,069 [Logger](CRITICAL): Uncaught exception
Traceback (most recent call last):
  File "./backseat_app/main.py", line 111, in <module>
    main()
  File "./backseat_app/main.py", line 107, in main
    backseat_app.spin()
  File "./backseat_app/main.py", line 75, in spin
    self.marl_processor.compute_new_heading()
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/processing.py", line 163, in compute_new_heading
    self.new_action = self.rl_tracking.newAction(self.target_address,-1,self.lrauv_pose,self.lrauv_depth,self.target_timestamp,new_range=False)
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/utils.py", line 931, in newAction
    self.action, self.target_predictions = self.agent_controller.get_action_and_predictions(
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/../jaxagent/jaxagent/production_agent.py", line 43, in get_action_and_predictions
    pred = tracker.update_and_predict(
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/../jaxagent/jaxagent/tracking/tracker.py", line 44, in update_and_predict
    ranges[i] = np.sqrt(r**2 - (pos[2] - depth)**2)
TypeError: unsupported operand type(s) for ** or pow(): 'NoneType' and 'int'
